mesonbuild/minstall.py

```diff
@@ -409,7 +410,6 @@ def do_copyfile(self, from_file: str, to_file: str,
             dirmaker, outdir = makedirs
             # Create dirs if needed
             dirmaker.makedirs(outdir, exist_ok=True)
-        self.log(f'Installing {from_file} to {outdir}')
```

Comment:
Hmm actually maybe we should just move it to above makedirs? If we move it above `if os.path.exists` then we change what gets logged, in the case of `--only-changed`.

Reply:
good catch, fixed now.

I ended up having to duplicate the log message in the `if` and `else` branches - in the case of `os.path.exists()` we still need to log before `self.remove()` in case that throws an error. couldn't find a good way to streamline this without temp variables but they ended up just convoluting things further.